src/oc-id/app/views/email_verify_mailer/email_verify.text.erb

```diff
@@ -0,0 +1,16 @@
+Hello <%= @user.first_name %>,
+
+It looks like you've just tried to change the email address associated with
+your Chef user account. If you did not initiate this request, please ignore
+this email. If you are indeed trying to update your account, follow
+this link to confirm your new email address.
+
+<%= email_profile_url(
```

Comment:
And in case you are confused reading this...  email_profile_url is a dynamically generated method based on <subroute>_<route>_url(...) because...  magic...